### PR TITLE
MIG product_sequence

### PR DESCRIPTION
**Pull request by angelmoya**

### CODE REVIEW

File: product_sequence/README.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+A module that adds sequence to the product.
+====================================================================
+
+This module allows to associate a sequence to the product reference.
+The reference (default code) is unique (SQL constraint) and required.
+Note: This module is incompatible with nan_product_sequence.
```
Comment (pedrobaeza) [inline product_sequence/README.rst]: You can put a contributors section with your name here. There's a suggested template for OCA modules here: https://github.com/OCA/maintainers-tools/blob/master/template/module/README.rst

File: product_sequence/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (c) 2015 Domatix (http://domatix.com)
+#                       Angel Moua <angel.moya@domatix.com>
```
Comment (pedrobaeza) [inline product_sequence/models/__init__.py]: s/Moua/Moya (I think you want to put your name correctly :wink:)

**Comment by pedrobaeza** [comment]
As in https://github.com/OCA/product-attribute/pull/33, I would remove here the constraint about code uniqueness.

File: product_sequence/README.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
+A module that adds sequence to the product.
+====================================================================
```
Comment (pedrobaeza) [inline product_sequence/README.rst]: This line should be of the same length of the title

File: product_sequence/README.rst (hunk 1 above)
Comment (yvaucher) [inline product_sequence/README.rst]: This is done

**Comment by yvaucher** [comment]
:+1:

File: product_sequence/README.rst [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+This module allows to associate a sequence to the product reference.
+The reference (default code) is unique (SQL constraint) and required.
+Note: This module is incompatible with nan_product_sequence.
```
Comment (pedrobaeza) [inline product_sequence/README.rst]: This module is no longer on 8, so you can remove it.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by pedrobaeza** [comment]
I merge, and we will resolve later the split of the module in 2.

**Pull request merged**